src/apps/statements/tests/storingStatements/queriables/getMetadataFromStatement/getDurationMetadata.test.ts

```diff
@@ -0,0 +1,32 @@
+import * as assert from 'assert';
+
+import { getDurationMetadata } from '../../../../service/storeStatements/queriables/getMetadataFromStatement/getDurationMetadata';
+import { statementDefaults } from './fixtures/statements.fixture';
+
+describe('Retrieve duration metadata from statement', () => {
+  it('should return result duration statement', () => {
+      const expectedEmptyMetadata = false;
```

Comment:
I think this block is tabbed out too far, I'm not sure why the linter hasn't pointed that out.